### PR TITLE
July 2024 update

### PR DESCRIPTION
**Pull request by g7morris**
Release notes:

* All services below have an updated README.md, new software at the latest version,  new SHA etc. along with having been tested via processes outlined here if there should be any questions / concerns: https://gist.github.com/g7morris/05818f8da4bac3c129063f1478f9b078
  * `activemq`
  * `fits` 
    * `fits-servlet` - updated software
    * `log4j` - updated software
  * `handle` 
  * `tomcat`

* Updated service README.mds to latest version only.
  * `fcrepo`
  * `fits`
  * `mariadb`
  * `nginx`
  * `postgresql`
  * `solr`

---

@nigelgbanks and @joecorall Once reviewed and approved, either one or both of you can merge in but if there are too many commits, I'm happy to squash them down to one if need be. Let me know please and thanks in advance!

### CODE REVIEW

File: postgresql/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # PostgreSQL
 
-Docker image for [PostgreSQL] version 15.1
+Docker image for [PostgreSQL] version 16.3
```
Comment (nigelgbanks) [inline postgresql/README.md:3]: I got 16.2 when I checked.

```
/ $ postgres --version
postgres (PostgreSQL) 16.2
```

File: nginx/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Nginx
 
-Docker image for [Nginx] version 1.22.1 and [FPM] version 8.1.15.
+Docker image for [Nginx] version 1.22.4 and [FPM] version 8.3.8.
```
Comment (nigelgbanks) [inline nginx/README.md:3]: I get the following.

```
/var/www $ nginx -version
nginx version: nginx/1.24.0
```

and

```
/var/www $ php-fpm83 -version
PHP 8.3.8 (fpm-fcgi) (built: Jun  6 2024 17:04:22)
Copyright (c) The PHP Group
Zend Engine v4.3.8, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.8, Copyright (c), by Zend Technologies
```

**Review by nigelgbanks — CHANGES_REQUESTED** [review]
Other than 2 version numbers being off, it looks good. I've tested locally. Cheers.

File: nginx/README.md (hunk 2 above)
Comment (g7morris) [reply nginx/README.md]: Agreed. I think this was a typo on my part. Thank you for spotting that. Fixed now. 

* Reran `make bake && make up`
* `docker compose exec drupal bash`
* `nginx -version`
* `php-fpm83-version`

Same results as above.

File: postgresql/README.md (hunk 1 above)
Comment (g7morris) [reply postgresql/README.md:3]: Hmmmm I'm not getting the same results from a fresh build?

* Reran `make clean && make bake && make up`

**Check 1:**

 ```bash
docker run --rm -d --name postgresql islandora/postgresql

➜  isle-buildkit git:(july-2024-update) docker exec -ti postgresql psql -U root postgres
psql (16.3)
Type "help" for help.

postgres=#
```

Check 2:
```bash
docker exec -ti postgresql bash

6f89c38c08af:/# postgres --version

postgres (PostgreSQL) 16.3
```

File: nginx/README.md (hunk 2 above)
Comment (g7morris) [reply nginx/README.md]: Resolving this comment thread as fixed.

File: postgresql/README.md (hunk 1 above)
Comment (nigelgbanks) [reply postgresql/README.md:3]: My local build matches the version from the latest release:

```
make bake BUILDER=desktop-linux TARGET=postgresql 2&>/dev/null && docker run --rm -ti -u 1 --entrypoint psql islandora/postgresql:local --version
psql (PostgreSQL) 16.2
```

So, I obliterated all my local builds and then built without access to the cache on DockerHub and got what you have. Which is worrying, I'll open up a ticket to investigate this cache poisoning issue.

**Review by nigelgbanks — APPROVED** [review]

**Pull request merged**